cfpb/consumerfinance.gov
# Store model name in elasticsearch for filterable lists

It turns out, on our enforcement and event pages, we limit the result set to specific class type, this allows us to implement the same logic with elasticsearch.

---




## Additions

- model_class field to elasticsearch index
- Additional filter on ES results to limit them to a specific class when applicable


## Removals

-


## Changes

-


## How to test this PR

1. Search enforcement actions and ensure only EnforcementActionPage results return.


## Screenshots


## Notes and todos

-


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
